src/java/com/zimbra/graphql/resolvers/impl/AccountResolver.java

```diff
@@ -59,6 +60,16 @@ public AccountInfo accountInfoGet(@GraphQLRootContext RequestContext context)
         return accountRepository.accountInfoGet(context);
     }
 
+    @GraphQLQuery(description="Retrieve info")
+    public GetInfoResponse infoGet(@GraphQLRootContext RequestContext context,
```

Comment:
Use `info` - as described above you're querying for info, so no need to declare `Get`.

Also, for consistency let the context be the final parameter.